sktime/datatypes/tests/test_vectorize.py

```diff
@@ -0,0 +1,321 @@
+# -*- coding: utf-8 -*-
+"""Testing vectorization via VectorizedDF."""
+
+__author__ = ["fkiraly"]
+
+import numpy as np
+import pytest
+
+from sktime.datatypes import MTYPE_REGISTER, SCITYPE_REGISTER
+from sktime.datatypes._check import check_is_mtype
+from sktime.datatypes._examples import get_examples
+from sktime.datatypes._vectorize import VectorizedDF
+from sktime.utils._testing.deep_equals import deep_equals
+
+SCITYPES = ["Panel", "Hierarchical"]
+
+
+def _get_all_mtypes_for_scitype(scitype):
+    """Return list of all mtypes for scitype.
+
+    Parameters
+    ----------
+    scitype : str - scitype
+
+    Returns
+    -------
+    mtypes : list of str - list of mtypes for scitype
+    """
+    if scitype not in [s[0] for s in SCITYPE_REGISTER]:
+        raise RuntimeError(scitype + " is not in the SCITYPE_REGISTER")
+    mtypes = [key[0] for key in MTYPE_REGISTER if key[1] == scitype]
+
+    if len(mtypes) == 0:
+        # if there are no mtypes, this must have been reached by mistake/bug
+        raise RuntimeError("no mtypes defined for scitype " + scitype)
+
+    return mtypes
+
+
+def _generate_scitype_mtype_combinations():
+    """Return scitype/mtype tuples for pytest_generate_tests.
+
+    Fixtures parameterized
```

Comment:
Why is this called Fixtures parameterized? Should just be Parameters

Reply:
well, it's not parameters, since these are no function parameters.
Not sure how to describe it, since it produces fixture combinations.
Can also remove it?